src/main/kotlin/no/nb/mlt/wls/order/payloads/SynqOrderPayload.kt

```diff
@@ -32,13 +31,8 @@ data class SynqOrderPayload(
         val quantityOrdered: Double
     )
 
-    enum class SynqOrderType(private val type: String) {
-        STANDARD("Standard");
-
-        @JsonValue
-        override fun toString(): String {
-            return type
-        }
+    enum class SynqOrderType {
+        STANDARD
```

Comment:
This is likely a merge error, as this code is required for SynQ to properly accept our payloads

Reply:
Oh yeah might have missed that, thanks for finding it